README.md

```diff
@@ -18,49 +18,37 @@
+Enriching HTTP clients with capabilities related to **security**, **performance** or **resilience** is a common need but it is also an error-prone and sometimes complex task.
 
-## Examples
+🎯 **HttpExt** has been built with one goal in mind: helping you focus on your apps' features instead of the transport layer's boilerplate and matters... and without any trade-off.
 
-Checkout the [demo app workspace](./apps/sandbox) for a concrete example.
-
-## Ecosystem
+- 🅰️ **HttpExt** is **Angular-ready** and makes [interceptors](https://angular.io/api/common/http/HttpInterceptor) implementation safe and easy,
+- 🐍 **HttpExt** is **fully reactive** and based on [RxJS](https://rxjs-dev.firebaseapp.com/),
+- 🔋 **HttpExt** has **batteries included** as it comes with some useful plugins,
+- 📈 **HttpExt** is **progressive** because you can start using it without having to rewrite all your HTTP calls,
+- 🧱 **HttpExt** is **easily extendable** as you can create and share your own plugins.
 
-This project is a monorepo that includes the following packages.
+# How It Works
 
-| Name                                          | Description    | Goal                     | Size                                                                   |
-| --------------------------------------------- | -------------- | ------------------------ | ---------------------------------------------------------------------- |
-| [@http-ext/core](./libs/core)                 | Core           | Plugins handler          | ![cost](https://badgen.net/bundlephobia/minzip/@http-ext/core)         |
-| [@http-ext/angular](./libs/angular)           | Angular module | HttpClient compatibility | ![cost](https://badgen.net/bundlephobia/minzip/@http-ext/angular)      |
-| [@http-ext/plugin-auth](./libs/plugin-auth)   | Auth plugin    | Authenticate requests    | ![cost](https://badgen.net/bundlephobia/minzip/@http-ext/plugin-auth)  |
-| [@http-ext/plugin-cache](./libs/plugin-cache) | Cache plugin   | Cache HTTP resources     | ![cost](https://badgen.net/bundlephobia/minzip/@http-ext/plugin-cache) |
-| [@http-ext/plugin-retry](./libs/plugin-retry) | Retry plugin   | Retry failed requests    | ![cost](https://badgen.net/bundlephobia/minzip/@http-ext/plugin-retry) |
+The main building block is the plugin. A plugin is a simple object that lets you intercept network communications and control or transform them easily. Like an _HttpInterceptor_ a plugin may transform outgoing request and the response stream as well before passing it to the next plugin. The library comes with a built-in [plugin collection](#built-in-plugins) to provide useful behaviors for your apps and to tackle the need to rewrite redundant logic between projects. It's also possible to [create your own plugin](#custom-plugin) for handling custom behaviors.
```

Comment:
The last two sentences should go in the **Motivation** section, or being deleted if it's redundant.

Reply:
Finally I prefer with it, it's more descriptive.

Reply:
@edbzn  
I was also wondering which place was best for this. 
But finally, this is probably the best place for now as it helps understand and gives links to dive deeper for those who want to.